Sparkle/SUHost.m

```diff
@@ -62,6 +62,17 @@ - (NSString *)bundlePath
     return [self.bundle bundlePath];
 }
 
+- (BOOL)allowsAutomaticUpdates
+{
+    // Can we automatically update in the background without bugging the user (e.g, with a administrator password prompt)?
+    BOOL isWritableAtBundle = [[NSFileManager defaultManager] isWritableFileAtPath:self.bundlePath];
+    
+    // Does the developer want us to disable automatic updates?
+    NSNumber *developerAllowsAutomaticUpdates = [self objectForInfoDictionaryKey:SUAllowsAutomaticUpdatesKey];
```

Comment:
Maybe reverse order would be slightly faster, avoiding disk access in some cases?

```
if (developerAllowsAutomaticUpdates) return developerAllowsAutomaticUpdates.boolValue;
return …isWritableFileAtPath…;
```